python/lsst/ci/hsc/validate.py

```diff
@@ -6,6 +6,16 @@
 import functools
 from lsst.pex.logging import getDefaultLog
 from lsst.daf.persistence import Butler
+from SCons import Action
+
+class callAction(Action.FunctionAction):
```

Comment:
Commentary describing what's going on here would be helpful. I don't mind if that appears in this class, in the `makeScons` method, or both, but some explanatory text is necessary.